spec/controllers/sms_broker/incoming_messages_controller_spec.rb

```diff
@@ -56,9 +56,9 @@ module SmsBroker
 
       it 'ensures correct encoding on incoming text' do
         pending('this test is broken')
```

Comment:
This test is still pending. Still not sure how to fix that (have `Invalid query parameters: Non UTF-8 value:`) error